tests/conftest.py

```diff
@@ -14,11 +14,11 @@
 @contextmanager
 def temporary_package_file() -> Iterator[Path]:
     url = "https://conda.anaconda.org/conda-forge/linux-64/xtensor-0.16.1-0.tar.bz2"
-    with requests.get(url, stream=True) as response:
-        with NamedTemporaryFile() as file:
-            with open(file.name, "wb") as fp:
-                shutil.copyfileobj(response.raw, fp)
-            yield Path(file.name)
+    xtensor_raw_download = requests.get(url, stream=True).raw
+    with NamedTemporaryFile() as file:
+        with open(file.name, "wb") as fp:
+            shutil.copyfileobj(xtensor_raw_download, fp)
+        yield Path(file.name)
```

Comment:
Any reason you prefer this style?

Reply:
precommit was complaining a bit, see [here](https://github.com/mamba-org/quetz-client/actions/runs/4206689364/jobs/7300508630#step:4:131)